README.md

```diff
@@ -31,6 +31,17 @@ Gunrock is featured on NVIDIA's list of [GPU Accelerated Libraries](https://deve
 </tr>
 </tbody></table>
 
+## Quickstart
+```
+git clone --recursive https://github.com/gunrock/gunrock/
+cd gunrock
```

Comment:
Would like the opinion of @sgpyc and @neoblizz : I usually make a parallel directory to "gunrock" called "gunrock-build" (instead of putting build within gunrock). Also "gunrock-docs" and "gunrock-io". @bkj is the norm to do a build within the source directory?

Reply:
Different people do it differently I think.  I was following the install instructions from [the docker install](https://github.com/gunrock/gunrock/blob/master/docker/ubuntu16.04-cuda9.0/dockerfile)

Reply:
I create a build directory inside of gunrock, and this is how Jenkins tests it as well. @sgpyc creates a parallel directory to gunrock. :)

One thing though is that do you we wanna put this under docs instead of individual folders within our main repo? For example, one page on "gunrock's docker guide"?

Reply:
Either way is fine, but it's probably good to be consistent.  I think `build` inside the repo is more standard than a parallel folder?  I don't have a strong opinion though.

Not sure I understand your second question?

Reply:
"more standard" is my preference.